rna_design/rna_synthesizer_n.py

```diff
@@ -0,0 +1,316 @@
+#!/usr/bin/env python
+
+
+from itertools import tee
+from itertools import izip
+import logging
+
+from sklearn.linear_model import SGDClassifier
+
+from eden.converter.fasta import fasta_to_sequence
+from eden.converter.rna.rnashapes import rnashapes_to_eden
+from eden.util import fit
+from eden.graph import Vectorizer
+from eden.modifier.seq import seq_to_seq
+from eden.modifier.seq import shuffle_modifier
+
+from rna_design.constraint_extractor import ConstraintExtractor
+from rna_designer import AntaRNAv117Designer
+
+
+logger = logging.getLogger(__name__)
+
+
+class Initializer():
+
```

Comment:
Documentation is ongoing. There has been loys of structural changes so documentation comes the last.